pkg/webhook/ingress/validating.go

```diff
@@ -56,74 +59,80 @@ func (w *webhook) GetPath() string {
 	return "/validating-ingress"
 }
 
-type handler struct{}
+type handler struct {
+	Log logr.Logger
+}
 
 func Handler() capsulewebhook.Handler {
-	return &handler{}
+	return &handler{Log: ctrl.Log.WithName("controllers").WithName("Tenant")}
 }
 
 func (r *handler) OnCreate(client client.Client, decoder *admission.Decoder) capsulewebhook.Func {
+
 	return func(ctx context.Context, req admission.Request) admission.Response {
-		i, err := r.ingressFromRequest(req, decoder)
+		ingress, err := r.ingressFromRequest(req, decoder)
 		if err != nil {
 			return admission.Errored(http.StatusBadRequest, err)
 		}
 
-		return r.validateIngress(ctx, client, i)
+		return r.validateIngress(ctx, client, ingress)
 	}
 }
 
 func (r *handler) OnUpdate(client client.Client, decoder *admission.Decoder) capsulewebhook.Func {
+
 	return func(ctx context.Context, req admission.Request) admission.Response {
-		i, err := r.ingressFromRequest(req, decoder)
+		ingress, err := r.ingressFromRequest(req, decoder)
 		if err != nil {
 			return admission.Errored(http.StatusBadRequest, err)
 		}
 
-		return r.validateIngress(ctx, client, i)
+		return r.validateIngress(ctx, client, ingress)
 	}
 }
 
 func (r *handler) OnDelete(client client.Client, decoder *admission.Decoder) capsulewebhook.Func {
+
```

Comment:
I saw this new line at the beginning of every function in the files you edited: is this an IDE linting option?

Reply:
Yes, just a personal preference. I can remove it.

Reply:
You're not the first, indeed! :)

We're not documenting this in the coding style documented for Capsule, but we can create an issue to document this, too.

Please, let's keep the current style.